article/models.py

```diff
@@ -1,12 +1,17 @@
 from django.db import models
 # Create your models here.
-
+class Treat(models.Model):
+    title =  models.CharField(max_length=20,verbose_name="Title")
 class Article(models.Model):
     author = models.ForeignKey("auth.User",on_delete = models.CASCADE,verbose_name = "Author")
     title = models.CharField(max_length = 50,verbose_name = "Title")
-    content = models.TextField()
```

Comment:
I think TextField() will be bettern than charField as content is usually more than 256 characters.